spec/features/loan_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature 'Loans', js: true do
+  let!(:documents) { create(:document, :with_file) }
+  let!(:loan_type) { create(:loan_type, :with_documents) }
+    
+  feature 'Go to loans page' do
+    scenario 'Exist an loan' do
+      visit "/loans"
+
+      expect(page).to have_text('Loans')
```

Comment:
@ericksonmc the scenario is saying that it checks that a loan exists, but it's not. It's just checking for the page title. Something is wrong here.